configure: error: Please reinstall the libzip distribution

I got error below running ```docker-compose up -d``` :
```
configure: error: Please reinstall the libzip distribution
ERROR: Service 'e2' failed to build: The command '/bin/sh -c apt-get update  && apt-get install -y         libpng-dev         libjpeg-dev         libmcrypt-dev         unzip  && rm -rf /var/lib/apt/lists/*  && docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr  && docker-php-ext-install gd mbstring mysqli pdo_mysql zip' returned a non-zero code: 1

```
 So to fix this error i have added       ```libzip-dev \```  to dockerFile.

--- a/Dockerfile
+++ b/Dockerfile
@@ -2,8 +2,9 @@ FROM php:7-apache
 
 ARG VERSION=3254
 
-RUN apt-get update \
+RUN apt-get update \ 
  && apt-get install -y \
+        libzip-dev \
         libpng-dev \
         libjpeg-dev \
         libmcrypt-dev \

--- a/Dockerfile
+++ b/Dockerfile
@@ -2,7 +2,7 @@ FROM php:7-apache
 
 ARG VERSION=3254
 
-RUN apt-get update \ 
+RUN apt-get update \
  && apt-get install -y \
         libzip-dev \
         libpng-dev \

